Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -402,7 +402,7 @@ - (void)loadView
     if (fullScreenControlDesignEnabled)
```

Comment:
While you're here, can you rename this local to have the word "alternate" at the front?